.drone.star

```diff
@@ -631,81 +672,137 @@ def phptests(testType):
+					if nameLength > maxLength:
+						print("Error: generated phpunit stage name of length", nameLength, "is not supported. The maximum length is " + str(maxLength) + ".", name)
+						errorFound = True
+
+					if (filesExternalType == ''):
+						# for the regular unit test runs, the clover coverage results are in a file named like:
+						# autotest-clover-sqlite.xml
+						coverageFileNameStart = 'autotest'
+						extraCoverageRenameCommand = []
+						extraCoverage = False
+					else:
+						# for the files-external unit test runs, the clover coverage results are in 2 files named like:
+						# autotest-external-clover-sqlite.xml
+						# autotest-external-clover-sqlite-samba.xml
+						coverageFileNameStart = 'autotest-external'
+						extraCoverageRenameCommand = [
+							'mv tests/output/coverage/%s-clover-%s-%s.xml tests/output/coverage/clover-%s-%s.xml' % (coverageFileNameStart, getDbName(db), externalType, name, externalType)
+						]
+						extraCoverage = True
+
+					result = {
+						'kind': 'pipeline',
+                        'type': 'docker',
+                        'name': name,
+                        'workspace' : {
+                            'base': '/var/www/owncloud',
+                            'path': 'server/apps/%s' % config['app']
+                        },
+                        'steps':
+                            installCore('daily-master-qa', db, False) +
+                            installApp(phpVersion) +
+                            installExtraApps(phpVersion, params['extraApps']) +
+                            setupServerAndApp(phpVersion, params['logLevel']) +
+                            setupCeph(params['cephS3']) +
+                            setupScality(params['scalityS3']) +
+                            params['extraSetup'] +
+                        [
+                            {
+                                'name': '%s-tests' % testType,
+                                'image': 'owncloudci/php:%s' % phpVersion,
+                                'pull': 'always',
+                                'environment': params['extraEnvironment'],
+                                'commands': params['extraCommandsBeforeTestRun'] + [
+                                    command
+                                ]
+                            }
+                        ] + params['extraTeardown'],
+                        'services':
+                            databaseService(db) +
+                            cephService(params['cephS3']) +
+                            scalityService(params['scalityS3']) +
+                            params['extraServices'],
+                        'depends_on': [],
+                        'trigger': {
+                            'ref': [
+                                'refs/pull/**',
+                                'refs/tags/**'
+                            ]
+                        }
+                    }
+
+					if params['coverage']:
```

Comment:
And the indent changes again here. I haven't looked in my IDE. Maybe the block of code above line 740 has spaces instead of tabs.